frontend/pages/hosts/ManageHostsPage/_styles.scss

```
@@ -123,4 +128,90 @@
       }
     }
   }
+
+  &__team-dropdown {
+    border: 0 !important;
+    position: relative;
+
+    :hover {
+      cursor: pointer !important;
+    }
+
+    &.is-focused {
+      .Select-control {
+        border: 0 !important;
+        height: 32px;
+      }
+    }
+
+    .Select-menu-outer {
+      position: absolute;
+      width: 100%;
```

```suggestion
      width: 330px;
```

I'm suggesting an intermediate step to improving the dropdown styles here.

We explicitly define the `width` of the "Status" dropdown in the same file here: https://github.com/fleetdm/fleet/blob/main/frontend/pages/hosts/ManageHostsPage/_styles.scss#L118